CONTRIBUTING.md

```diff
@@ -18,13 +18,27 @@ git remote add upstream https://github.com/slack-ruby/slack-ruby-bot.git
 
 ## Bundle Install and Test
 
-Ensure that you can build the project and run tests.
+First make sure you've installed all default gems
 
 ```
 bundle install
-bundle exec rake
 ```
 
+To run the full specs suite you'll need to install one of the optional gems: `async-websocket`, `faye-websocket` or `celluloid-io`.
+
+```
+gem install async-websocket -v 0.8.0
```

Comment:
We don't want to recommend a global async-websocket install, it should go into the Gemfile of the application using the library (which is already in the docs). The correct(er) way is to `export CONCURRENCY=async-websocket` before running `bundle install`, and it will install the network lib.

Reply:
I'm not sure what to put here.

I wouldn't rely on other app Gemfile while developing this gem since you might not have such app at all in your dev environment. The instruction to install the `async-websocket` is not a recommendation but an example of a minimum setup to run the specs. Do you want to highlight that it's just an example? Or you have something more specific in mind?

Reply:
But this is a CONTRIBUTING to this project, and our Gemfile includes [this](https://github.com/slack-ruby/slack-ruby-bot/blob/master/Gemfile#L7). Remove this paragraph and say to `export CONCURRENCY=async-websocket` _before_ `bundle install` and running tests.